mac-dubbo-admin打包编译报错

mac java-version
java version "12.0.1" 2019-04-16
Java(TM) SE Runtime Environment (build 12.0.1+12)
Java HotSpot(TM) 64-Bit Server VM (build 12.0.1+12, mixed mode, sharing)

构建

[ERROR] Failed to execute goal on project dubbo-admin-server: Could not resolve dependencies for project org.apache:dubbo-admin-server:jar:0.1: The following artifacts could not be resolved: org.apache.dubbo:dubbo:jar:2.7.2-SNAPSHOT, org.apache.dubbo:dubbo-serialization-kryo:jar:2.7.2-SNAPSHOT: Could not find artifact org.apache.dubbo:dubbo:jar:2.7.2-SNAPSHOT in apache.snapshots.https (https://repository.apache.org/content/repositories/snapshots) -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/DependencyResolutionException
[ERROR] 
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <goals> -rf :dubbo-admin-server
<img width="1677" alt="dubbo-admin-编译报错" src="https://user-images.githubusercontent.com/3424856/59236762-2479fc00-8c2a-11e9-80ec-8292a4a532c2.png">

--- a/pom.xml
+++ b/pom.xml
@@ -55,8 +55,8 @@
 
 	<properties>
 		<main.basedir>${project.basedir}</main.basedir>
-        <commons-lang3-version>3.7</commons-lang3-version>
-		<dubbo-version>2.7.2-SNAPSHOT</dubbo-version>
+                <commons-lang3-version>3.7</commons-lang3-version>
+		<dubbo-version>2.7.3-SNAPSHOT</dubbo-version>
 		<curator-version>2.12.0</curator-version>
 		<curator-test-version>4.1.0</curator-test-version>
 		<fastjson-version>1.2.46</fastjson-version>
@@ -64,7 +64,7 @@
 		<netty-version>4.1.30.Final</netty-version>
 		<jacoco-version>0.8.2</jacoco-version>
 		<jedis-version>2.9.0</jedis-version>
-        <apollo-version>1.2.0</apollo-version>
+                <apollo-version>1.2.0</apollo-version>
 		<snakeyaml-version>1.24</snakeyaml-version>
 		<maven-checkstyle-plugin-version>3.0.0</maven-checkstyle-plugin-version>
 		<spring-boot-version>2.1.4.RELEASE</spring-boot-version>

